### PR TITLE
Install cloud SDK/tools

### PR DESCRIPTION
**Pull request by chrissng**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -29,6 +29,17 @@ RUN mvn dependency:copy-dependencies && \
     # Purge local maven cache
     rm -rf /root/.m2
 
+# Install cloud tools
+ARG CLOUD_SDK_VERSION=360.0.0-0
+ARG AWS_CLI_VERSION=1.20.63
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add - && \
+    apt-get update -y && \
+    apt-get install -y google-cloud-sdk=${CLOUD_SDK_VERSION} google-cloud-sdk-app-engine-python=${CLOUD_SDK_VERSION} && \
```
Comment (sweatybridge) [inline Dockerfile:38]: I think no need to install `google-cloud-sdk-app-engine-python` component?
```suggestion
    apt-get install -y google-cloud-sdk=${CLOUD_SDK_VERSION} && \
    rm -rf /var/lib/apt/lists/* && \
```

also should purge the cache afterwards

File: Dockerfile [hunk 2]
```diff
@@ -29,6 +29,17 @@ RUN mvn dependency:copy-dependencies && \
...
+    gcloud --version && \
+    pip install awscli==${AWS_CLI_VERSION} && \
```
Comment (sweatybridge) [inline Dockerfile:40]: Shall we make a `requirements.txt` for all pip packages? It will benefit from dependabot checks.

File: Dockerfile (hunk 1 above)
Comment (chrissng) [reply Dockerfile:38]: Included in case they needed it. im not sure if they need it also

File: Dockerfile (hunk 2 above)
Comment (chrissng) [reply Dockerfile:41]: Good idea

Comment (chrissng) [reply Dockerfile]: Added a requirements.txt file

**Pull request closed without merging**